Makefile

```diff
@@ -1,8 +1,16 @@
-all: create-test-certs build
+.ONESHELL:
 
-create-test-certs:
+all: build
+
+generate-sample-certs:
 	make -C test/auth
 
+test-curl-server:
+	curl --cert test/auth/certs/sample-vmd-client-crt.pem	\
+		--key test/auth/certs/sample-vmd-client-key.pem		\
+		--cacert test/auth/certs/sample-ca-crt.pem			\
+		http://localhost:8000/
+
```

Comment:
a minor style issue but I'm not big fan of backslashes for line continuation running wild with various # of tabs and spaces after the last character of the line

Reply:
I'll correct this